URBNSwiftAlert/Classes/URBNSwAlertAction.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  URBNSwAlertAction.swift
+//  Pods
+//
+//  Created by Kevin Taniguchi on 5/23/17.
+//
+//
+
+import Foundation
+
+public enum URBNSwAlertActionType {
```

Comment:
We’re 100% swift here _(and I don’t think we’ll be pulling this into the legacy apps)_. I think I’d get rid of the URBNSw prefix everywhere